docs/en/operation/Admin-CLI.md

```diff
@@ -69,26 +54,39 @@ $ ./bin/alluxio fsadmin doctor configuration
 
 ### report
 
-// TODO(lu) detailed description about each of the subcommands.
-// such as the report capacity usage with filter tags.
-
 The `report` command provides Alluxio running cluster information. 
 
+If no argument is passed in, `report` will report the leader master, worker number, and capacity information.
+
 ```bash
-# Report cluster summary
 $ ./bin/alluxio fsadmin report
-#
-# Report worker capacity information
+```
+
+`report capacity` will report Alluxio cluster capacity informatin of different ranges: 
```

Comment:
... will report Alluxio cluster capacity information for different subsets of workers

Reply:
DONE